detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -145,10 +158,10 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
       DD4hep::Geometry::Tube layer_base(bcalInnerR,bcalOuterR,layerThickness*0.5);
       DD4hep::Geometry::SubtractionSolid layer_subtracted;
       { // put this in extra block to limit scope
-	const double thisPositionZ = bcalCentreZ + referencePosition + layerThickness*0.5;
-	const DD4hep::Geometry::Position thisBPPosition( std::tan(-fullCrossingAngle) * thisPositionZ, 0.0, 0.0);
-	const DD4hep::Geometry::Transform3D thisBPTransform( incomingBeamPipeRotation, thisBPPosition );
-	layer_subtracted = DD4hep::Geometry::SubtractionSolid(layer_base, incomingBeamPipe, thisBPTransform);
+        const double thisPositionZ = bcalCentreZ + referencePosition + layerThickness*0.5;
```

Comment:
You can, for example, use interactive rebase to edit the commit that includes the whitespace change.
See: http://stackoverflow.com/a/29950959